### DISCUSSION TITLE
New Submarines and Tier Balance

### DISCUSSION THREAD

**Discussion by GaoDong64** [FakeFishGames/Barotrauma#11596] [Ideas and suggestions]
Hello! I think I echo a large population of the player base when I say people are always hoping for new submarines, but I wanted to point out an option in submarine design. For the most part, all Tier 1 subs are small, and Tier 2 and 3 subs get progressively larger. While there is a talent in-game that allows any submarine to be upgradable to Tier 3, this does little to “reward” players of a small crew beyond allowing them to progress to the endgame. In essence, an optional talent is necessary for small crews to “finish” the campaign without pulling their hair out trying to manage a submarine too large. 

With this in mind, if we do get additional submarines, I’d like to suggest adding ships of different sizes for different tiers.

For example, the largest tier 1 subs are for a crew of 4-6. It may do well to add one large Tier 1 Sub for a crew of 5-9, and add smaller Tier 2 and 3 subs designed for crews of 3-6. 

This way, larger crews can still experience Tier 1 subs, and small crews can experience a state-of-the-art small sub with advanced systems.

**Comment by aeondez** [comment]
I agree strongly with this. 

My core group of 3 players just finished a Vanilla campaign right before the 1.0 release. 

We went Barsuk > Camel > Humpback > Winterhalter

We actually didn't know about the Mechanic talent and thought we were forced to switch to a T3 to complete the game. We had no choice but to hire bots just to keep the sub in repair and to man all the guns.

I get the assumption that bigger = better, but the typical group size rarely changes for me, and we usually play with 2-6 people. Most T3 subs require 7-16 people to man properly, meaning I have to resort to custom subs just to finish the game without bots. Bots have their place, but should never be mandatory. 

A small T3 sub meant for a small crew would be welcome.

**Comment by duarm** [comment]
Typhoon 2 is the only choice for <4 people. I think Iron Storm is a placeholder to patch the problem for now,  The idea is for us to switch subs, hence the tier system, but they just don't have enough submarines for now. The Iron Storm perk is contradictory to that goal, and kinda useless, they should eventually change it.
